### DISCUSSION TITLE
flink-doris-connect 无法本地环境写入doris集群

### DISCUSSION THREAD

**Discussion by levisLi** [apache/doris#13567] [A - General / Q&A]
通过FE访问http://ll03:8030/api/backends?is_alive=true接口获取的be的ip地址是内网环境的ip，本地无法访问，请问这个怎么处理转换为外网的ip了
Backend Info:{"backends":[{"ip":"10.240.11.225","http_port":8050,"is_alive":true}｝

**Comment by levisLi** [comment]
flink连接doris时会报如下错误，因为访问的是内网IP
Connect to 10.240.11.100:8050 [/10.240.11.100] failed: Connection timed out: connect

**Reply by caoliang-web** [reply]
这个可能目前没办法转化

**Reply by zy-kkk** [reply]
这个目前无法转换，flink通过fe的http端口获取到的是be的内网ip，你需要让你的flink程序运行在能和be内网ip连通的环境当中
